packages/server/lib/util/ci_provider.js

```diff
@@ -135,10 +139,12 @@ const _providerCiParams = () => {
       'BUILD_REPOSITORY_URI',
     ]),
     bamboo: extract([
-      'bamboo.resultsUrl',
-      'bamboo.buildNumber',
-      'bamboo.buildResultsUrl',
-      'bamboo.planRepository.repositoryUrl',
+      'bamboo_buildNumber',
```

Comment:
Why did you use `_` instead of `.` to get the variables. They are documented on atlassian's site as `bamboo.buildNumber` not `bamboo_buildNumber`. https://confluence.atlassian.com/bamboo/bamboo-variables-289277087.html#Bamboovariables-Build-specificvariables

Reply:
In the Using variables in bash section, it mentions that all system variables are exported as bash variables. I could add both forms to cover all the bases, but I didn't want to bloat the bamboo section

Reply:
Wow, I skimmed the whole page and couldn't see this, what a small important mention. 

>Bamboo variables are exported as bash shell variables. All full stops (periods) are converted to underscores. For example, the variable bamboo.my.variable is $bamboo_my_variable in bash.

Thanks, this is what I was looking for.

Reply:
Bamboo's certainly been an adventure